_variables.yml

```diff
@@ -0,0 +1,23 @@
+grass:
+  name: GRASS
+  version: 8.4
```

Comment:
Why version? We would need to keep it updated. I would remove it.

Reply:
I thought it would be useful to have the latest version as a variable so if it is mentioned we can do a quick update.

Reply:
Let's remove it.

Reply:
I think it will be good to have variable with the current GRASS version, so we can be version specific in the tutorials.